README.md

```diff
@@ -866,6 +868,29 @@ MyEntity:
 
 > 📝 `MISTER` in `<enum(Civility::MISTER)>` refers to a constant defined in the `Civility` enum class, not to a constant's value ('mister' string for instance).
 
+## Api-Platform
+
+### OpenApi / Swagger
+The PhpEnums library provides an `Elao\Enum\Bridge\ApiPlatform\Core\JsonSchema\Type\ElaoEnumType` to generate a OpenApi (formally Swagger) documentation based on your enums.
+
+To use it, you need to decorate the `api_platform.json_schema.type_factory` service.
+
+```yml
+# config/services.yaml
+services:
+    # symfony <= 5.0
+    Elao\Enum\Bridge\ApiPlatform\Core\JsonSchema\Type\ElaoEnumType:
```

Comment:
Let's do this directly from the bundle extension if the decorated factory exists, wdyt?
You could write this in the [`services.xml`](https://github.com/Elao/PhpEnums/blob/master/src/Bridge/Symfony/Bundle/Resources/config/services.xml) and drop the definition if the class doesn't exist.

Is there any drawback/potential harm registering this service automatically without the ability to disable it?

Reply:
I dont think so. The only case would be if you want to write your own decorator for this lib which is unlikely imho.

I will add it to the services.xml.

Reply:
Nice, thank you.

You can now remove this wiring from the documentation (you can keep the section & first sentence, but mention everything should be wired for you when using the bundle)